core/frontend/src/components/vehiclesetup/configuration/compass/CompassDisplay.vue

```diff
@@ -49,8 +49,8 @@ export default Vue.extend({
       required: true,
     },
     colors: {
-      type: Array as PropType<string[]>,
-      default: () => ['green', 'blue', 'purple'],
+      type: Object as PropType<Dictionary<string>>,
+      default: () => ({}),
```

Comment:
Is this empty default okay?

Reply:
@Williangalvani any comment on that ?

Reply:
I found a different issue while looking at it, I'll come back to it